static/src/stylesheets/module/tailor/_tailor-survey-overlay.scss

```diff
@@ -0,0 +1,128 @@
+.tailor-survey-overlay,
+.tailor-survey-background {
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	z-index: 1050;
+	position: fixed;
+	display: block;
+}
+
+.tailor-survey-overlay .tailor-survey-background {
+	background: #000000;
+	opacity: .5;
+	z-index: 1;
+}
+
+.tailor-survey-overlay .tailor-survey-container {
+	width: 380px;
+	height: 300px;
+	position: absolute;
+	margin: auto;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	background-color: $guardian-brand;
+	z-index: 2;
+
+	.tailor-survey-container-header,
+	.tailor-survey-proceed-button {
+		font-family: $f-sans-serif-text;
+		font-weight: 900;
+		font-size: 18px;
+	}
+
+	.tailor-survey-container-header {
+		height: 42px;
+		width: 100%;
+		background-color: $news-support-1;
+		color: $neutral-1;
+		line-height: 42px;
+		padding-left: 8px;
+		box-sizing: border-box;
+
+		.inline-marque-36 {
+			position: absolute;
+			top: 3px;
+			right: 8px;
+		}
+	}
+
+	h1,
+	p {
+		color: $neutral-6;
+		font-size: 22px;
+		line-height: 25px;
+		margin: 12px 8px;
+		font-family: $f-serif-headline;
+	}
+
+	h1 {
+		font-weight: 900;
+	}
+
+	p {
+		font-weight: 100;
+	}
+
+	.tailor-survey-proceed-button {
+		color: $guardian-brand;
+		background-color: $news-support-1;
+		border: 0;
+		border-radius: 20px;
+	    padding: 4px 8px 2px 12px;
+	    margin-top: 12px;
+	    margin-left: 8px;
+	    height: 42px;
+
+	    .inline-chevron-right {
+	    	fill: $guardian-brand;
+	    	margin-left: 12px;
+	    }
+
+	    .inline-chevron-right svg {
+			height: 18px;
+			width: 9px;
+			margin-bottom: -4px;
+	    }
+
+	    &:hover {
+	    	background-color: #4bc6df;
+	    }
+
+	    &:focus {
+	    	outline:0;
+	    }
+	}
+
+	.tailor-survey-close-button {
+		border: 1px solid $neutral-6;
+		background-color: $guardian-brand;
+		height: 42px;
+		width: 42px;
+		border-radius: 21px;
+		position: absolute;
+		right: 8px;
+		bottom: 8px;
+	    padding-top: 9px;
+
+		.inline-cross svg {
+	    	fill: $neutral-6;
+	    }
+
+	    &:hover {
+	    	border-color: #ffffff;
+
+			.inline-cross svg {
+				fill: #ffffff;
+			}
+	    }
+
+	    &:focus {
```

Comment:
Could you add a focus style, maybe even just duplicate of hover for keyboard users?